test/GhSyncTests/AdoTests.cs

```diff
@@ -1,18 +1,49 @@
+namespace gh_sync.Tests;
+
 using Xunit;
 using gh_sync;
-using Octokit;
 using System;
+using Octokit;
 using System.Threading.Tasks;
+using System.Linq;
 using Microsoft.TeamFoundation.WorkItemTracking.WebApi.Models;
+using Microsoft.Extensions.DependencyInjection;
 
-
-public class AdoTests
+public record class AdoTests(MockStartup Startup) : IClassFixture<MockStartup>
 {
+    private IAdo MockAdo => Startup.Services.GetRequiredService<IAdo>();
+    Ado Ado = new Ado();
+
+    [Fact]
+    public async Task Given_null_id_throw_exception()
```

Comment:
These method names don't seem to follow C# coding conventions.
```suggestion
    /// <summary>
    ///     Checks that passing <c>null</c> as an issue or work item ID throws an exception.
    /// </summary>
    public async Task ThrowsExceptionWhenGivenNullId()
```

Reply:
I've updated the tests to use CamelCase as well as include summaries.